### ISSUE TITLE
DATA statements of type STRING not allowing multiple lines

### ISSUE THREAD

**Issue by jonnypencils** [neilsf/xc-basic3#180]
It seems subsequent DATA statements after the first one are ignored for type STRING

`DIM rn$(9) AS STRING * 20 @roomname 

roomname:
DATA AS STRING * 20 "study", "hallway", "corner of room"
DATA AS STRING * 20 "stairway", "garden","by the lake"
DATA AS STRING * 20 "the old tree", "fallen down wall", "garden shed"

FOR I AS BYTE = 0 to 8
PRINT rn$(I)
NEXT I`

rn$(3) and above are blank.

![image](https://user-images.githubusercontent.com/83018447/169997607-0dc8a32d-cecf-41a2-ba1c-ef982b41b8ce.png)

NB This behaviour is _not_ seen for numeric data types, they all work as expected.

**Comment by Jest0r** [comment]
Looks like a "DC.B" is added to the end of each string data line. Here a comparison between 10 strings defined on one line vs. 3 lines:
```
DATA as string * 12 "kitchen", "living room"
DATA as string * 12 "bedroom", "balcony", "bathroom"
DATA as string * 12 "room6", "room7", "room8", "room9", "room1"
```
vs
```
DATA as string * 12 "kitchen", "living room", "bedroom", "balcony", "bathroom", "room6", "room7", "room8", "room9", "room1"
```
![image](https://user-images.githubusercontent.com/22813091/169999378-3abd5086-bad0-44a4-969d-a3f56c9ce42a.png)

**Comment by neilsf** [comment]
Fixed in v3.0.6
